_data/recommendations.yaml

```diff
@@ -1,40 +0,0 @@
----
-    People:
```

Comment:
is it better to centralize these data files? i guess they won't change frequently.. but might be easier if we want to have the data in multiple places

Reply:
I checked it in initially because of concerns about tool rot ten years down the line. It occurs to me now, though, I could fork Eleventy and also back the podcast up.

Reply:
Re: Centralization, I think it's better to keep the data close to where it will actually be used, following the episode model. If we need to use them in multiple places, we can just pull the hosts in to wherever.

For example, with episodes, we include them in the home page (`index.njk`) like this:

          {%- for episode in collections.episodes -%}

But we also have that same line in `rss.xml.njk'.

Reply:
Re: Centralization, I think it's better to keep the data close to where it will actually be used, following the episode model. If we need to use them in multiple places, we can just pull the hosts in to wherever.

For example, with episodes, we include them in the home page (`index.njk`) like this:

          {%- for episode in collections.episodes -%}

But we also have that same line in `rss.xml.njk'.